.gitignore

```diff
@@ -1 +1,3 @@
 *.gem
+nclude-dependencies
```

Comment:
can you fix this?

Reply:
In general the lock file is not included in the plugins(since it depends on build env). Thats why i didnt include. I checked with other plugins too. Its the same